test/helpers/ledger.ts

```diff
@@ -0,0 +1,6 @@
+import { Transaction } from "@services/ledger/schema"
+
+export const getAccountTransactionsCount = (account: string, query = {}) => {
```

Comment:
missing `async` marker I think... its returning a `Promise`

Reply:
Funnily enough this also worked without the `async` marker. Adding it for completeness though.